hack/make-rules/test-cmd-util.sh

```diff
@@ -1629,15 +1629,32 @@ run_non_native_resource_tests() {
   # Make sure it's gone
   kube::test::get_object_assert foos "{{range.items}}{{$id_field}}:{{end}}" ''
 
+  # Test that we can watch the resource. Use coproc to read from stdout asynchronously.
+  coproc kubectl "${kube_flags[@]}" get bars --request-timeout=1m --watch-only -o name
```

Comment:
It's a bash-only feature, but the hashbang at the top of the script specifies bash anyway:

https://www.gnu.org/software/bash/manual/html_node/Coprocesses.html